src/Chapter4.hs

```diff
@@ -114,25 +114,26 @@ As always, try to guess the output first! And don't forget to insert
 the output in here:
 
 >>> :k Char
-
+Char :: *
 >>> :k Bool
-
+Bool :: *
 >>> :k [Int]
-
+[Int] :: *
 >>> :k []
-
+[] :: *->*
 >>> :k (->)
-
+(->) :: *->*->*
 >>> :k Either
-
+Either :: *->*->*
```

Comment:
I see that spaces are removed between `*` and `->` here. To pass doctests, you need to insert the line exactly as GHCi displays, because doctest compare with the GHCi output